projects/plugins/jetpack/modules/search/instant-search/components/gridicon/index.jsx

```diff
@@ -56,7 +56,7 @@ class Gridicon extends Component {
 			case 'gridicons-comment':
 				return <title>{ __( 'Matching comment.', 'jetpack' ) }</title>;
 			case 'gridicons-cross':
-				return <title>{ __( 'Close search results', 'jetpack' ) }</title>;
+				return <title>{ __( 'Close search interface', 'jetpack' ) }</title>;
```

Comment:
"interface" is pretty technical language - shall we stick with "results"?

Reply:
I agree that "interface" is too technical to be used here. The original intent of my change was to make the cross more explicitly indicate overlay closure. In this case, we're not merely closing "search results", but the entire overlay including various search controls that would be reset in such a closure.

Maybe "window" or "screen" would be better?

Reply:
I think "window" is better but would still lean towards describing the content with "search results".